docs/concepts/reftypes.mdx

```diff
@@ -5,33 +5,46 @@ sidebar_position: 1
+enabling the movement of artifacts and their dependencies between environments.
 
 ## Implementing Reference Types in OCI artifacts
 
-The concept of linking or relating OCI artifacts is achieved using the concept of reference types, where one artifact can be set as a subject to another. In this context, "subject" refers to the target of a reference.
+The concept of linking or relating OCI artifacts is achieved using the concept
+of reference types, where one artifact can be set as a subject to another.
+In this context, "subject" refers to the target of a reference.
 
 ### The Art of Associating Artifacts
 
-Consider a typical cloud-native environment with countless OCI artifacts. Often, it's imperative to manage relationships between them—say, an OCI artifact's signatures, or metadata about an artifact. This is where reference types come into the picture.
+Consider a typical cloud-native environment with countless OCI artifacts.
+Often, it's imperative to manage relationships between them—say, an OCI artifact's
+signatures, or metadata about an artifact. This is where reference types come into the picture.
 
-A reference type enables an OCI artifact to refer to another, creating a defined link or association between them. This association can be for various purposes, such as versioning, updates, signatures, or dependencies.
+A reference type enables an OCI artifact to refer to another, creating a defined
+link or association between them. This association can be for various purposes,
+such as versioning, updates, signatures, or dependencies.
 
 ![OCI artifact with refererrs pointing to the artifact](/img/concepts/reftypes.png)
 
 ## Creating a Reference Artifact
 
-OCI Artifacts may be stored in a content-addressable storage system or OCI Layout directory. For the purpose of this guide, we'll use the OCI Distribution Specification to create and list reference types.
+OCI Artifacts may be stored in a content-addressable storage system or OCI Layout directory.
+For the purpose of this guide, we'll use the OCI Distribution Specification to
+create and list reference types.
 
 The creation of a reference artifact is a two-step process:
 
-1. **Step One**: Put an artifact, say Artifact A, using the PUT Manifest and blob upload APIs if the artifact has blob content.
+1. **Step One**: Put an artifact, say Artifact A, using the PUT Manifest and
+blob upload APIs if the artifact has blob content.
     - The prerequisite for a reference type is that the subject artifact must exist in the repository.
 
-2. **Step Two**: Put another artifact, say Artifact B, which refers to Artifact A. In this case, Artifact A is the 'subject' of Artifact B.
-    - This is done using the [push workflow][push-workflow] in the distribution specification.
+2. **Step Two**: Put another artifact, say Artifact B, which refers to Artifact A.
+In this case, Artifact A is the 'subject' of Artifact B.
+> [!NOTE]  
```

Comment:
The `> [!NOTE]` mark only works in GitHub rendered pages. Apparently, the docusaurus does not render it properly.

<img width="716" alt="image" src="https://github.com/oras-project/oras-www/assets/32161882/16943597-ab65-4d6b-91ee-b207388610c7">

Reply:
I've updated it to use docusarus style :::tip but also moved that content out of the note and instead made a call out the fact that subject does not need to exist.